app/src/main/java/com/duckduckgo/app/browser/omnibar/animations/BrowserLottieTrackersAnimatorHelper.kt

```diff
@@ -456,6 +462,19 @@ class BrowserLottieTrackersAnimatorHelper @Inject constructor(
         }
     }
 
+    private fun getVisualDesignAnimationRawRes(
+        logos: List<TrackerLogo>,
+        theme: AppTheme,
+    ): Int {
+        val trackers = logos.size
+        return when {
+            trackers == 1 -> if (theme.isLightModeEnabled()) R.raw.light_trackers_1_visual_updates else R.raw.dark_trackers_1_visual_updates
+            trackers == 2 -> if (theme.isLightModeEnabled()) R.raw.light_trackers_2_visual_updates else R.raw.dark_trackers_2_visual_updates
+            trackers >= 3 -> if (theme.isLightModeEnabled()) R.raw.light_trackers_visual_updates else R.raw.dark_trackers_visual_updates
+            else -> TODO()
```

Comment:
Missing case to handle (or not, but let's clean up the TODO()).

Reply:
This one still needs addressing in some way.

Reply:
will show a default and be done with it